liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/BasicSupport.java

```diff
@@ -228,6 +233,11 @@ protected void init() throws MojoExecutionException, MojoFailureException {
             
             log.info(MessageFormat.format(messages.getString("info.variable.set"), "serverDirectory", serverDirectory));
             
+            // Set shared direcory
+            File sharedDirectory = new File(userDirectory, "shared");
+            
+            sharedResourcesDirectory = new File(sharedDirectory, "resources");
```

Comment:
Do we need to worry about whether this directory exists or not? I'm wondering if before copying over the lib.index.cache file if there needs to be a check. Or does the ant task take care of that?

Reply:
Added sharedResourcesDirectory.mkdirs() if sharedResourcesDirectory doesn't exist.